### ISSUE TITLE
Maintain log of documentation changes and enable subscription to doc updates

### ISSUE THREAD

**Issue by Yerkon** [angular/angular#22489]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

I came across to [ChangeLog ](https://angular.io/guide/change-log)page. This page was updated last time in **2017-03-13**, although after this there were many changes. Is this page no longer updated(supported)? 

## Expected behavior

It will be nice have some **notification/subscription** feature to actual documentation changes. When new docs(guides) is added/removed/updated etc.,  how to get informed about that?

**Comment by jenniferfell** [comment]
Thanks for your suggestion. I just recently learned about that changelog file. 

I opened this issue about resolving the first part of your request: "Where can someone learn about changes to documentation?" https://github.com/angular/angular/issues/23615

That issue doesn't include your second suggestion, which is being able to subscribe to something that will notify you of documentation changes. I'm keeping your issue open because it nicely makes both points. 

Another related issue: https://github.com/angular/angular/issues/17409
